.github/workflows/pkgci_regression_test_cpu.yml

```diff
@@ -58,20 +58,36 @@ jobs:
             experimental/regression_suite
 
       # Out of tree tests
-      - name: Checking out external TestSuite repository
+      - name: Install git lfs
+        run: |
+          sudo apt-get install git-lfs
+          git lfs install
```

Comment:
This seems to be failing on a postsubmit build bot: https://github.com/openxla/iree/actions/runs/8442874639/job/23125596377#step:8:16
`E: Unable to locate package git-lfs`

Might need that installed in a Dockerfile or just directly on the self hosted runners. @pzread maybe you can advise on that? I think this is running on a GCP machine?

Reply:
Succeeded on a retry... scary...

Reply:
And failed on another run: https://github.com/openxla/iree/actions/runs/8443729776/job/23128477271 :/
Might revert this until I'm confident it will keep working.